translate/src/modules/editor/components/TranslationLength.tsx

```diff
@@ -5,13 +5,7 @@ import { getPlainMessage } from '~/utils/message';
 
 import './TranslationLength.css';
 
-/**
- * Shows translation length vs. original string length, or countdown.
- *
- * Countdown is currently only supported for LANG strings, which use special
- * syntax in the comment to define maximum translation length. MAX_LENGTH
- * is provided for strings without HTML tags, so they need to be stripped.
```

Comment:
It's surprising no other format supports this feature.

Reply:
I'm not surprised, but I also wonder if we should keep this around in some forms (how, given no other format has a comment/directive for max length?). We have discussed this before, from semantic comments in Fluent, to lately for CSV (LSP support).

Reply:
I don't think it's feasible to keep it around. Maybe we file a generic issue and reference this PR so that we can reuse the code.